torch/distributed/algorithms/model_averaging/averagers.py

```diff
@@ -15,11 +14,14 @@ class ModelAverager(ABC):
                        will be used. (default: ``None``)
     """
 
-    def __init__(self, process_group=None):
+    def __init__(self, process_group=None, comm_memory_efficient=False):
         self.process_group = (
             process_group if process_group is not None else dist.group.WORLD
         )
         self.step = 0
+        # comm_memory_efficient is False, use a faster communication but more memory average strategy
+        # comm_memory_efficient is True, use a slower communication but less memory average strategy
```

Comment:
Unsure what "memory average" means, do you mean memory efficient?

Reply:
yes, according @wayi1 review, will delete comm_memory_efficient argument

Reply:
delete done